validator/testcases/regex.py

```diff
@@ -558,6 +558,67 @@ def tests(self):
             compat_type='error')
 
 
+@register_generator
+class Gecko47RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 47 updates."""
+
+    VERSION = FX47_DEFINITION
+
+    def tests(self):
+        yield self.get_test(
+            r'\bnsIX509CertDB\b',
+            'Most methods in nsIX509CertDB had their unused arguments removed.',
+            (
+                'Most methods in nsIX509CertDB had their unused arguments '
+                'removed. See %s for more information.' % BUGZILLA_BUG % 1241646
+            ),
+            log_function=self.err.warning,
+            compat_type='error')
+
+        yield self.get_test(
+            r'\b(listTokens|listSlots)\b',
+            'listTokens() and listSlots() now return nsISimpleEnumerator instead of nsIEnumerator.',
+            (
+                'listTokens() and listSlots() now return nsISimpleEnumerator instead '
+                'of nsIEnumerator. See %s for more information.' % BUGZILLA_BUG % 1220237
```

Comment:
Right but I mostly implemented along https://github.com/mozilla/addons-server/issues/2228 which says "compatibility error" and that's what we do here. I suppose they need to be checked for sure but it could be that there are some false positives.

Checking wether or not the code already uses `listTokens` / `listSlots` correctly is very complicated and might not even work imho.